### DISCUSSION TITLE
ZigBee Rule: On ZbReceived Rule wildcard

### DISCUSSION THREAD

**Discussion by pierpaolopalazzo** [arendst/Tasmota#20132] [Q&A] [answered]
Hello,

with several similar sensors, i would like to have something like:

Rule1 ON ZbReceived#*#Humidity DO WebSend [192.168.69.252] /zb.php?z=%zbdevice%&d=h-%value% ENDON

or

Rule1 ON ZbReceived#*#* DO WebSend [192.168.69.252] /zb.php?z=%zbdevice%&d=%value%&c=%zbcluster% ENDON

i.e. a wildcard on devices and/or clusters

is it possible? can it be implemented?

**Comment by sfromis** [comment, marked as answer]
Read the docs to find the wildcard already available..... https://tasmota.github.io/docs/Rules/#rule-trigger

**Comment by pierpaolopalazzo** [comment]
wow.

thanks a lot.

i really overlooked that section although i was deeply searching for it

it works like a charm:

15:32:41.315 RUL: ZBRECEIVED#?#HUMIDITY performs "WebSend [192.168.69.252] /zb.php?d=0xA09B-h-51.4"
